projects/fuzztest-example/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+# Extend with oss-fuzz settings. To be upsteamed?
+cat $SRC/setup_configs.sh >> fuzztest/bazel/setup_configs.sh
+
+# Compile gfuzztests
+cd fuzztest/codelab
+compile_gfuzztests.sh
```

Comment:
Is it not possible to link fuzztest against a target like we do with our other engines?

Reply:
The approach here is to link against the current fuzz engines, so build fuzztest fuzzers using the compatibility mode (https://github.com/google/fuzztest/blob/878a7d8c60baf95bfbadb4722da7e8f832f529e2/bazel/setup_configs.sh#L90) -- in this case we support both libfuzzer and centipede.